iothub/service/src/Common/Exceptions/ServerErrorException.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Azure.Devices.Common.Exceptions
     /// The exception that is thrown when the IoT Hub returned an internal service error.
     /// </summary>
     /// <remarks>
-    /// This exception typically means the IoT Hub service has encountered an unexpected error and is usually transient. Please review the <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-troubleshoot-error-500xxx-internal-errors">500xxx Internal errors</see> guide for more information. The best course of action is to retry your operation after some time. By default, the SDK will utilize the <see cref="ExponentialBackoff"/> retry strategy.
```

Comment:
We have service SDK exception referring to a device SDK type, which fails my local build. Also, the service SDK doesn't have a retry strategy, right?

Reply:
No retry strategy that I know of.

Reply:
Correct. None of our service clients (ServiceClient, RegistryManager, JobClient, etc) have any retry logic. Good catch here